person.rb

```diff
@@ -0,0 +1,21 @@
+class Person
+  attr_accessor :name, :age
+  attr_reader :id
+
+  def initialize(id, age:, name: 'unknown', parent_permission: true)
+    @id = id
+    @name = name
+    @age = age
+    @parent_permission = parent_permission
+  end
+
+  def can_use_services?
+    @age || parent_permission
+  end
+
+  private
+
+  def of_age
+    @age >= 18
+  end
```

Comment:
- [ ] **[OPTIONAL]** Per convention this type of method that returns boolean should have a question mark.

  ```
  def of_age?
    @age >= 18
  end
  ```